lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSignupCard.js

```diff
@@ -0,0 +1,93 @@
+import Button from './Button';
+
+const React = require('react');
+
+/**
+ * Signup Card Component
+ * <SignupCard />
+ */
+class CampaignSignupCard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      actionDisabled: false,
+      isSignedUp: false,
+      isLoading: false,
+    };
+
+    this.signup = this.signup.bind(this);
+  }
+
+  signup() {
```

Comment:
Feels a little weird to declare the same named method as the one passed via props, but I kinda see why. Fine for now I think, but just noting :)